### DISCUSSION TITLE
How do I use schema from outside of respective tenant?

### DISCUSSION THREAD

**Discussion by surajbeston** [lorinkoz/django-pgschemas#90] [Q&A] [answered]
I'm in a use case where I'm looking for django-tenant's [schema_context](https://django-tenants.readthedocs.io/en/latest/use.html#schema_context) decorator to execute code from outside of schema. 

I couldn't find anything by myself. Can anyone give me a clue for how to do that? Thanks!

**Comment by surajbeston** [comment]
I actually found a way to do it referencing the middleware:

> DomainModel = get_domain_model()
>domain = DomainModel.objects.select_related("tenant").get(domain=domain, folder=prefix)
>activate(domain.tenant)


Now I can run code as being inside the tenant.
I'm not sure if its the best way but it works.

**Comment by lorinkoz** [comment, marked as answer]
That's right. Any of these should work:

```
activate(tenant)


with tenant:
    # do something
```
